src/App.js

```diff
@@ -1,47 +1,66 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import styled from 'styled-components'
 import useAudio from './hooks/useAudio'
 import PlayPage from './pages/PlayPage'
+import playlist from './data/playlist.json'
+import usePlaylist from './hooks/usePlayList'
 
 function App() {
-  const url =
-    'https://audio-ssl.itunes.apple.com/itunes-assets/Music/12/f7/c9/mzm.hyfizvmg.aac.p.m4a'
-
-  const urlLong =
-    'https://api.freeplaymusic.com/media/downloadable/files/link_samples/media/volume/edits/Soundtrack_Rock_Volume_5/e/a/eastbound_60.mp3'
-  const { songUrl, toggle, stop, playing, duration } = useAudio(urlLong)
-  const [answers, setAnswers] = useState([
-    { title: 'Bulls on Parade', right: true },
-    { title: 'Bomb Track', wrong: true },
-    { title: 'People of the Sun', wrong: true },
-  ])
   const [showAnswer, setShowAnswer] = useState(false)
 
+  // usePlaylist takes a list of Objects including an iTunes trackId as id.
+  // it checkes if the Id response valid and then retuns a random url from the list and matching random answers
+  // with next song we can trigger to get another url
+  const { newUrl, answers, nextSong } = usePlaylist(playlist)
+
+  // useAudio hold and controll the audio element
+  const { songUrl, toggle, stop, playing, duration } = useAudio()
+
+  const [loaded, setLoaded] = useState(false)
+  const [newAnswers, setNewAnswers] = useState(answers)
+
+  //Checks if necessary data are loaded
+  useEffect(() => {
+    answers && setLoaded(true)
+  }, [answers])
```

Comment:
It looks to me like you can spare a state and an effect. 

```suggestion
  const [newAnswers, setNewAnswers] = useState(answers)
const loaded = !!answers
```